shared/version/version.go

```diff
@@ -10,5 +13,20 @@ var buildDate = "Moments ago"
 
 // GetVersion returns the version string of this build.
 func GetVersion() string {
+	// if doing a local build, these values are not interpolated
+	if gitCommit == "{STABLE_GIT_COMMIT}" {
+		commit, err := exec.Command("git", "rev-parse", "HEAD").Output()
+		if err != nil {
+			log.Fatal(err)
+		}
+		gitCommit = strings.TrimRight(string(commit), "\r\n")
+	}
+	if buildDate == "{DATE}" {
+		now, err := exec.Command("date","-R","-u").Output()
```

Comment:
You dont need to execute an external command to find the current date and time. You can just use the time package

Reply:
Ok. That makes total sense. There is no RFC2822 in the time package. I can format it or just use one of the existing formats.